packages/opentelemetry-sdk-trace-base/test/common/Span.test.ts

```diff
@@ -1222,5 +1241,18 @@ describe('Span', () => {
         });
       });
     });
+
+    describe('when initial attributes are specified', () => {
+      it('should store specified attributes', () => {
```

Comment:
Test description doesn't really match. Seems it verifies that initial attributes are optional.
Maybe add one which actually has initial attributes (if it doesn't exist yet).

Reply:
Yea I fixed it.